Makefile

```diff
@@ -1,13 +1,17 @@
-build: build-cedar-14 build-heroku-16
+build: build-cedar-14 build-heroku-16 build-heroku-18
 
 build-cedar-14:
-	@echo "Building Nginx in docker for cedar-14..."
+	@echo "Building nginx in Docker for cedar-14..."
 	@docker run -v $(shell pwd):/buildpack --rm -it -e "STACK=cedar-14" -w /buildpack heroku/cedar:14 scripts/build_nginx /buildpack/bin/nginx-cedar-14
 
 build-heroku-16:
-	@echo "Building Nginx in docker for heroku-16..."
+	@echo "Building nginx in Docker for heroku-16..."
 	@docker run -v $(shell pwd):/buildpack --rm -it -e "STACK=heroku-16" -w /buildpack heroku/heroku:16-build scripts/build_nginx /buildpack/bin/nginx-heroku-16
 
+build-heroku-18:
+	@echo "Building nginx in Docker for heroku-18..."
+	@docker run -v $(shell pwd):/buildpack --rm -it -e "STACK=heroku-18" -e "NGINX_VERSION=1.13.12" -w /buildpack heroku/heroku:18-build scripts/build_nginx /buildpack/bin/nginx-heroku-18
```

Comment:
@tt Curious, why is heroku-18 the only entry to specify NGINX_VERSION?

Reply:
The build script specifies a default:

https://github.com/heroku/heroku-buildpack-nginx/blob/e64133f2f3ebccdcb8bbdca422294dfa6ecd6897/scripts/build_nginx#L12

However, that no longer compiles on Heroku-18 and upgrading across all stacks could have unintended consequences.